sdk/samples/iot/README.md

```diff
@@ -158,15 +160,125 @@ This section provides an overview of the different samples available to run and
 
 - *Executable:* `paho_iot_hub_pnp_component_sample`
 
-This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_sample.c) connects an IoT Plug and Play enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json). X509 self-certification is used.
+  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_sample.c) extends the IoT Hub Plug and Play Sample above to mimic a Temperature Controller.  This sample connects the IoT Plug and Play enabled device (the Temerature Controller) with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json).  If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect. X509 self-certification is used.
```

Comment:
```suggestion
  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_sample.c) extends the IoT Hub Plug and Play Sample above to mimic a Temperature Controller.  This sample connects the IoT Plug and Play enabled device (the Temperature Controller) with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json).  If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect. X509 self-certification is used.
```

Reply:
I always appreciate your mad typo catching skills.  I am very good at creating typos lol.  Thank you.

Reply:
don't put double spaces after periods.

Reply:
> I always appreciate your mad typo catching skills. I am very good at creating typos lol. Thank you.


Other than a quick scan/read of docs, I use tools like VS Code and browser extensions. No skills needed :P

Reply:
@ericwol-msft thanks.  That is a typing style I was taught in grade school (25+ year habit), so I may need your help hammering this single-space style into me until I get it right :)